utilities/test_suite/HIP/CMakeLists.txt

```diff
@@ -111,4 +115,23 @@ if(NIFTI_FOUND AND OpenCV_FOUND)
     target_link_libraries(Tensor_voxel_hip ${OpenCV_LIBS} -lturbojpeg -lrpp ${hip_LIBRARIES} pthread ${LINK_LIBRARY_LIST} hip::device ${NIFTI_PACKAGE_PREFIX}NIFTI::${NIFTI_PACKAGE_PREFIX}niftiio)
 else()
     message("-- ${Yellow}Warning: libniftiio must be installed to install ${PROJECT_NAME}/Tensor_voxel_hip successfully!${ColourReset}")
+endif()
+
+if(NOT libsnd_LIBS)
+    message("-- ${Yellow}Warning: libsndfile must be installed to install ${PROJECT_NAME}/Tensor_audio_hip successfully!${ColourReset}")
+else()
```

Comment:
@sampath1117 Please pull latest develop into this branch again, ideally the changes from this PR for the audio support flag should show - https://github.com/ROCm/rpp/pull/378/files#diff-60d74aa0b8e5fb394c2c48769bcd9be4c9481762d535f37eca99b957e0440f3fR122-R125

Reply:
Pls check other changes too in the test_suite, and test_suite/HIP CMake - https://github.com/ROCm/rpp/pull/378/files#diff-60d74aa0b8e5fb394c2c48769bcd9be4c9481762d535f37eca99b957e0440f3fR70-R77

Reply:
Added the required changes in HIP test suite CMake